### DISCUSSION TITLE
[RFC] support cls in supabase studio , allowing user to modify GRANT

### DISCUSSION THREAD

**Discussion by zhihengGet** [supabase/supabase#15604] [Feature Requests]
sometimes we want to restrict users from accessing an column, there are many ways to do this, but i think GRANT is the most intuitive one;

if supabase ui spports GRANT, it would be great

**Comment by bitnom** [comment]
> sometimes we want to restrict users from accessing an column, there are many ways to do this, but i think GRANT is the most intuitive one;
> 
> if supabase ui spports GRANT, it would be great

I think there is an issue or PR related to this but I'm having trouble relocating it.

**Reply by bitnom** [reply]
is here: https://github.com/supabase/supabase/pull/13745
